KinEcosystem/Info.plist

```diff
@@ -18,11 +18,6 @@
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
-	<key>NSAppTransportSecurity</key>
```

Comment:
this is the extension fix?

Reply:
No, that's removed because now that everything we load is https it's not needed anymore.